bindings/wasm/package.json

```diff
@@ -72,7 +72,9 @@
     "wasm-opt": "^1.3.0",
     "wasm-pack": "0.10.1",
     "webpack": "^5.67.0",
-    "webpack-cli": "^4.9.2"
+    "webpack-cli": "^4.9.2",
```

Comment:
What is `webpack-cli` used for? I didn't see any errors without it when running through the examples but might have missed something.

Reply:
No idea, I didn't add it. @eike-hass @abdulmth any ideas?

Reply:
I don't think we rely on it anymore, this is most likely left over from the previous example build. Should be fine to remove `webpack-cli` and `webpack`.